docs/tutorial/getting-started.md

```diff
@@ -1,3 +1,215 @@
+
+> Note: To exit the Pebble daemon, press Ctrl-C (which sends an "interrupt" signal to the process).
+
+## View, start and stop services
+
+While the Pebble daemon is running, you can view the status of services by running `pebble services`. Open another terminal tab, and run:
+
+```bash
+pebble services
+```
+
+You should see output similar to the following:
+
+```bash
+Service      Startup  Current  Since
+http-server  enabled  active   today at 11:30 UTC
+```
+
+Use `pebble stop <service1> <service2> ...` to stop one or more services. You can stop the running `http-server` service by running:
+
+```bash
+pebble stop http-server
+```
+
+You should get output similar to the following:
+
+```bash
+Service      Startup  Current   Since
+http-server  enabled  inactive  today at 11:33 UTC
+```
+
+Now the service `http-server` is stopped. If we run `curl localhost:8080` again, we get a "connection refused" error, which confirms the service is down.
+
+To start it again, run:
+
+```bash
+pebble start http-server
+```
+
+## Add a new layer
+
+Now let's add another layer containing a different service that is also an HTTP server. To create a new layer, run:
+
+```bash
+echo """\
+summary: Simple layer 2
+
+description: |
+    Yet another simple layer.
+
+services:
+    http-server-2:
+        override: replace
+        summary: demo http server 2
+        command: python3 -m http.server 8081
+        startup: enabled
+""" > $PEBBLE/layers/002-another-http-server.yaml
+```
+
+This creates another layer that also contains a single service (running a basic
+HTTP server using the Python `http.server` module listening on a different port 8081).
+
+Add the new layer to a Pebble plan by running:
+
+```bash
+pebble add layer1 $PEBBLE/layers/002-another-http-server.yaml
+```
+
+If the layer is added successfully, the above command should produce the following output:
+
+```bash
+Layer "layer1" added successfully from "/home/ubuntu/PEBBLE_HOME/layers/002-another-http-server.yaml"
+```
+
+Even though the service configuration has been updated with the newly added layer, the newly added services won't be automatically started. If we check the services:
+
+```bash
+pebble services
+```
+We can see that although the new service `http-server-2` has been added, it's still "inactive":
+
+```bash
+Service        Startup  Current   Since
+http-server    enabled  active    today at 11:41 UTC
+http-server-2  enabled  inactive  -
+```
+
+To bring the service state in sync with the new configuration, run `pebble replan`:
+
+```bash
+pebble replan
+```
+
+And you get output similar to:
+
+```bash
+2024-06-02T11:40:39Z INFO Service "http-server" already started.
+```
+
+Now if we check all the services again:
+
+```bash
+pebble services
+```
+
+We can see that the new HTTP server `http-server-2` defined in the newly added layer should have been started and be shown as "active":
+
+```bash
```

Comment:
Same comment about using the terminal output directive.
https://github.com/canonical/pebble/pull/421/files#r1635663342